cameronRomo/GameStack
# Feature/filter games

## Description

Adds a feature for users to filter games by genre or by platform.

## Related Issue

#2 

## Motivation and Context

As a user I would like the ability to filter down my options by genre and/or by platform so that I can compare more easily.

## How Has This Been Tested?

This PR has been tested with e2e cypress testing.

**Note:** The tests for the previous PR were not testing as expected, and this PR fixes those issues as well.

#### To manually test this PR:
  1. Run locally
  2. Confirm a list of genres on the left side of the screen
  3. Select a genre and note that it renders a list of games specific to that genre
  4. The selected genre should also be emphasised
  5. Confirm the existence of a "Platform" dropdown menue
  6. Select a platform and confirm a rerendering of games relative to that platform

## Screenshots (if appropriate):

![Screen Shot 2023-04-28 at 11 14 20 AM](https://user-images.githubusercontent.com/63012953/235326635-d26af456-978b-4d49-9e6a-5a1f55cf8fad.png)

## Types of changes




## Checklist:




- [x] UI changes are responsive.
